controllers/components/datasciencepipelines/datasciencepipelines_support.go

```diff
@@ -40,6 +40,10 @@ var (
 		"V2_DRIVER_IMAGE":                  "RELATED_IMAGE_ODH_ML_PIPELINES_DRIVER_IMAGE",
 		"V2_LAUNCHER_IMAGE":                "RELATED_IMAGE_ODH_ML_PIPELINES_LAUNCHER_IMAGE",
 		"IMAGESV2_ARGO_MLMDGRPC":           "RELATED_IMAGE_ODH_MLMD_GRPC_SERVER_IMAGE",
+		"IMAGES_PIPELINESRUNTIMEGENERIC":   "RELATED_IMAGE_ODH_ML_PIPELINES_RUNTIME_GENERIC_IMAGE",
+		"IMAGES_OAUTHPROXY":                "RELATED_IMAGE_ODH_ML_PIPELINES_OAUTHPROXY",
```

Comment:
Not sure if these dependencies should be referenced here. I see these images are already referenced in the csv and consumed by the DSP components
https://github.com/red-hat-data-services/RHOAI-Build-Config/blob/rhoai-2.16/bundle/manifests/rhods-operator.clusterserviceversion.yaml#L199-L201

Reply:
@HumairAK has an interesting point.
who is going to update these images SHA in CSV if any CVE is found.

Reply:
I've removed all the images that we are not buildling for now, and will introduce that as a separate requirement